README.md

```diff
@@ -50,39 +50,61 @@ const router = require('find-my-way')({
 
 <a name="on"></a>
 #### on(method, path, handler, [store])
-Register a new route, `store` is an object that you can access later inside the handler function.  
+Register a new route.
 ```js
-router.on('GET', '/', (req, res, params) => {
+router.on('GET', '/example', (req, res, params) => {
   // your code
 })
+```
+Last argument, `store` is used to pass an object that you can access later inside the handler function.
+```js
+router.on('GET', '/example', (req, res, params, store) => {
+  assert.equal(store, { message: 'hello world' })
+}, { message: 'hello world' })
+```
 
-// with store
-router.on('GET', '/store', (req, res, params, store) => {
-  // the store can be updated
-  assert.equal(store, { hello: 'world' })
-}, { hello: 'world' })
+#### on(methods[], path, handler, [store])
+Register a new route for each method specified in the `methods` array.
+It comes handy when you need to declare multiple routes with the same handler but different methods.
+```js
+router.on(['GET', 'POST'], '/example', (req, res, params) => {
+  // your code
+})
 ```
-If you want to register a **parametric** path, just use the *colon* before the parameter name, if you need a **wildcard** use the *star*.  
-*Remember that static routes are always inserted before parametric and wildcard.*
+
+<a name="supported-path-formats"></a>
+##### Supported path formats
+To register a **parametric** path, use the *colon* before the parameter name. For **wildcard** use the *star*.
+*Remember that static routes should always be inserted before parametric and wildcard.*
```

Comment:
This is not correct, static route should not be inserted before parametric and wildcard, they are.
We do that internally :)

Reply:
Oh, sorry for that, you're right...
I haven't done to much attention to that part before (https://github.com/delvedor/find-my-way/blob/master/node.js#L22-L30).